quarkchain/generated/.gitignore

```diff
@@ -0,0 +1,134 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
you don't need a full gitignore file here. just ignore all "*.py*

Reply:
done